### PR TITLE
Fix render songs for concrete album

### PR DESCRIPTION
**Pull request by OlegShynkarenko**

### CODE REVIEW

File: src/components/Albums/Albums.js [hunk 1]
```diff
@@ -15,12 +16,12 @@ export class AlbumsComponent {
...
       ".albums__card-ripple-effect"
     );
+    this.albumsContainer = this.mountPoint.querySelector(".albums__container");
   }
 
   initMaterial() {
     Array.from(this.albumRipplePoint).forEach(el => {
-      // eslint-disable-next-line no-new
-      new MDCRipple(el);
+      this.albumRipplePoint = new MDCRipple(el);
```
Comment (ihor4568) [inline src/components/Albums/Albums.js]: why did you do this? what is the purpose to assign each time to this smth in the loop?

File: src/components/Albums/Albums.js [hunk 2]
```diff
@@ -38,17 +39,30 @@ export class AlbumsComponent {
     );
   }
 
+  addEventsListeners() {
+    this.albumsContainer.addEventListener("click", e => {
+      e.path.forEach(node => {
+        if (node.classList && node.classList.contains("mdc-card-album")) {
```
Comment (ihor4568) [inline src/components/Albums/Albums.js]: please do not match material class names

File: src/components/Albums/Albums.js [hunk 3]
```diff
@@ -38,17 +39,30 @@ export class AlbumsComponent {
     );
   }
 
+  addEventsListeners() {
+    this.albumsContainer.addEventListener("click", e => {
+      e.path.forEach(node => {
```
Comment (ihor4568) [inline src/components/Albums/Albums.js]: is it really necessary? can we just check for e.target?

File: src/components/Albums/Albums.js (hunk 1 above)
Comment (WOLFRIEND) [reply src/components/Albums/Albums.js]: To assign each album ripple effect.

**Review by ihor4568 — APPROVED** [review]

**Pull request merged**